Ghidra/Features/Base/ghidra_scripts/ollvm_de_flattening.py

```diff
@@ -0,0 +1,460 @@
+## ###
+#  IP: GHIDRA
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+##
+# Ghidra script for OLLVM control flow de flattening
```

Comment:
Will this help message show in ghidra window? We might need a better usage description in that script manager window.